### ISSUE TITLE
upgrade casperjs causes tests to fail?

### ISSUE THREAD

**Issue by ultrasaurus** [18F/openopps-platform#1247]
I can reproduce the CI issue locally if I delete `node_modules` and do a fresh `npm install`

```
  1) "before all" hook

  0 passing (1s)

  1 failing

  1)  "before all" hook:
     Uncaught TypeError: 'undefined' is not an object (evaluating 'this.options.pageSettings.javascriptEnabled') (/Users/sarahallen/src/18F/openopps-platform/node_modules/casperjs/modules/casper.js:981)
```

**Comment by ultrasaurus** [comment]
suspect it is one of these

```
< ├── casperjs@1.1.0-beta3
---
> ├── casperjs@1.1.0-beta5
```

```
< ├── mocha-casperjs@0.5.4
---
> ├── mocha-casperjs@0.5.5
```

**Comment by ultrasaurus** [comment]
This seems to be the issue:  https://github.com/nathanboktae/mocha-casperjs/issues/80#issuecomment-183232733  but it should only fails with Phantom 2

**Comment by ultrasaurus** [comment]
this was addressed in 0.10.0 release
